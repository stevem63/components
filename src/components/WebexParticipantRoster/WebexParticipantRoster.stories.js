import React from 'react';
import {storiesOf} from '@storybook/react';

import jsonData from '../../data';
import {WebexJSONAdapter} from '../../adapters';

import WebexParticipantRoster from './WebexParticipantRoster';

import {WebexDataProvider} from '..';

// Setup for the stories
const stories = storiesOf('Webex Participant Roster', module);
const webexAdapter = new WebexJSONAdapter(jsonData);

// Stories
stories.add('default', () => (
  <WebexDataProvider adapter={webexAdapter}>
    <WebexParticipantRoster destination="default_membership" />
  </WebexDataProvider>
));
