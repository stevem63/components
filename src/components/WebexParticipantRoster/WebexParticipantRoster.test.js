import React from 'react';

import WebexParticipantRoster from './WebexParticipantRoster';

jest.mock('../hooks/usePerson');

describe('Webex Participant Roster component', () => {
  describe('snapshot', () => {
    test('matches snapshot of webex participant roster', () => {
      expect(shallow(<WebexParticipantRoster destination="default_membership" />)).toMatchSnapshot();
    });
  });
});
