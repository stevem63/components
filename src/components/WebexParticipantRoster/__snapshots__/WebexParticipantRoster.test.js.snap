// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Webex Participant Roster component snapshot matches snapshot of webex participant roster 1`] = `
<List
  active={null}
  className=""
  focus={null}
  focusFirst={true}
  focusFirstQuery=""
  focusQuery=""
  id={null}
  itemRole="listitem"
  onSelect={null}
  role="list"
  shouldFocusActive={false}
  shouldFocusInitial={true}
  shouldLoop={true}
  tabType="vertical"
  trackActive={true}
  type={null}
  wrap={false}
/>
`;
